internal/cloud/backend_apply.go

```diff
@@ -151,41 +153,69 @@ func (b *Cloud) opApply(stopCtx, cancelCtx context.Context, op *backend.Operatio
 		return r, err
 	}
 
-	logs, err := b.client.Applies.Logs(stopCtx, r.Apply.ID)
+	err = b.renderApplyLogs(stopCtx, r)
 	if err != nil {
-		return r, generalError("Failed to retrieve logs", err)
+		return r, err
+	}
+
+	return r, nil
+}
+
+func (b *Cloud) renderApplyLogs(ctx context.Context, run *tfe.Run) error {
+	logs, err := b.client.Applies.Logs(ctx, run.Apply.ID)
+	if err != nil {
+		return err
 	}
-	reader := bufio.NewReaderSize(logs, 64*1024)
 
+	reader := bufio.NewReaderSize(logs, 64*1024)
 	if b.CLI != nil {
 		skip := 0
 		for next := true; next; {
 			var l, line []byte
+			var err error
 
 			for isPrefix := true; isPrefix; {
 				l, isPrefix, err = reader.ReadLine()
 				if err != nil {
 					if err != io.EOF {
-						return r, generalError("Failed to read logs", err)
+						return generalError("Failed to read logs", err)
 					}
 					next = false
 				}
+
 				line = append(line, l...)
 			}
 
-			// Skip the first 3 lines to prevent duplicate output.
+			// Apply logs show the same Terraform info logs as shown in the plan logs
+			// (which contain version and os/arch information), we therefore skip to prevent duplicate output.
 			if skip < 3 {
 				skip++
 				continue
 			}
 
 			if next || len(line) > 0 {
-				b.CLI.Output(b.Colorize().Color(string(line)))
+				log := &jsonformat.JSONLog{}
+				if err := json.Unmarshal(line, log); err != nil || log == nil {
```

Comment:
💅 will log ever be `nil` ? I think not since you are assigning it the address of this struct